server/test/index.js

```diff
@@ -0,0 +1,23 @@
+const testFolder = './server/test';
```

Comment:
i think you don't need to all of these lines.
if you just require the files here that will be fine 
like : 
```
require("./Endpoints/router-test");
require("./Queries/query-test");
```

Reply:
ok bsh mohndes @knanahassouna1